apps/sms/views/inbox/js/inbox.js

```diff
@@ -39,6 +39,11 @@ var InboxView = {
   // Set to |true| when in edit mode
   inEditMode: false,
 
+  /**
+   * Indicates whether user should be notified about newly saved draft.
+   */
+  notifyAboutSavedDraft: false,
```

Comment:
I have mixed feelings, on one hand it shouldn't hurt since it's the first navigation step, but on the other hand it feels a bit tricky that ```beforeLeave``` can alter the intention of the caller and this step is the only one that can modify args. In the first days of Navigation.js I was thinking about deep frozen navigation args, but as I said I don't have strong opinion on that.